watcher/utils.go

```diff
@@ -0,0 +1,42 @@
+// maestro
+// https://github.com/topfreegames/maestro
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package watcher
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/topfreegames/maestro/models"
+)
+
+func dynamicDelta(
+	metricTrigger *models.ScalingPolicyMetricsTrigger,
```

Comment:
You could use ScalingPolicyMetricsTrigger as an interface actually, with the methods Metric() and Usage(). Or with the method DynamicDelta(). That way you won't need a switch/case statement.

Reply:
Will do